tests/tuplet_tests.js

```diff
@@ -2,482 +2,507 @@
  * VexFlow - Tuplet Tests
  * Copyright Mohit Muthanna 2010 <mohit@muthanna.com>
  */
-"use strict";
+
+/*
+eslint-disable
+no-undef,
+no-var,
+object-shorthand,
+func-names,
+wrap-iife,
+vars-on-top,
+ */
+
 VF.Test.Tuplet = (function() {
-  var Tuplet = {
-    Start: function() {
-      var runTests = VF.Test.runTests;
-      QUnit.module("Tuplet");
-      runTests("Simple Tuplet", Tuplet.simple);
-      runTests("Beamed Tuplet", Tuplet.beamed);
-      runTests("Ratioed Tuplet", Tuplet.ratio);
-      runTests("Bottom Tuplet", Tuplet.bottom);
-      runTests("Bottom Ratioed Tuplet", Tuplet.bottom_ratio);
-      runTests("Awkward Tuplet", Tuplet.awkward);
-      runTests("Complex Tuplet", Tuplet.complex);
-      runTests("Mixed Stem Direction Tuplet", Tuplet.mixedTop);
-      runTests("Mixed Stem Direction Bottom Tuplet", Tuplet.mixedBottom);
-      runTests("Nested Tuplets", Tuplet.nested);
-    },
+  // Ideally this would be using arrow syntax...
+  var set = function(propertyName) {
+    return function(value) {
+      return function(object) {
+        object[propertyName] = value;
+        return object;
+      };
+    };
+  };
```

Comment:
Unfortunately, ES5 syntax obfuscates how simple this is. I'd argue that there's nothing clever about this because it's just a curried function. 

Here's it refactored to use arrow functions: 

```javascript
var set = (propertyName) => (value) => (object) => {
  object[propertyName] = value;
  return object;
};
```

or with a [`curry` decorator:](https://lodash.com/docs#curry)

```javascript
var set = curry((propertyName, value, object) => {
  object[propertyName] = value;
  return object;
});